script/index.ts

```diff
@@ -37,8 +37,8 @@ function buildOptions() {
   return options;
 }
 
-function buildOptionsForLanguage(language: Language) {
-  let languageOptions = {};
+function buildOptionsForLanguage(language: Language, beautifiers: String[]) {
+  let languageOptions: any = {};
```

Comment:
Should be a more concrete interface. Do the types for Atom have any config related interfaces?

Reply:
Not from what I can tell, https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/atom/index.d.ts.

Reply:
Lame. Let's create our own interface, at least. Something simple.